about.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <!--font awsome link-->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>TriumphTribute</title>
+  </head>
+
+  <body>
+    <header>
+      <!--top nav-->
+      <nav class="topnav">
+        <div class="topnavcontainer">
+          <ul>
+            <li>
+              <a href="">
+                <i class="fa fa-facebook white" aria-hidden="true"></i>
+              </a>
+            </li>
+            <li>
+              <a href="">
+                <i class="fa fa-twitter white" aria-hidden="true"></i>
+              </a>
+            </li>
+            <li>
+              <a href="">English</a>
+            </li>
+            <li>
+              <a href="index.html">myPage</a>
+            </li>
+            <li>
+              <a href="">Logout</a>
+            </li>
+            <li>
+              <a href=""></a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+      <nav class="navbar">
+        <div class="padding">
+          <i id="bars" class="fa fa-bars absolute barsIcon" aria-hidden="true"></i>
+          <span id="close" class="absolute close">&times;</span>
+        </div>
+        <ul id="navlinks" class="navlinks padding">
+          <li>
+            <a class="mynav" href="index.html">Home</a>
+          </li>
+          <li>
+            <a class="mynav" href="#">About</a>
+          </li>
+          <li>
+            <a class="mynav" href="#">Program</a>
+          </li>
+          <li>
+            <a class="mynav" href="#">join</a>
+          </li>
+          <li>
+            <a class="mynav" href="#">sponser</a>
+          </li>
+          <li>
+            <a class="mynav" href="#">news</a>
+          </li>
+        </ul>
+        <!--desktop menu-->
+        <div class="deskMenu">
```

Comment:
[ ] Kindly add some margin so your navbar matches the one in the Figma design🤓
Your design 👇: 
![navhis](https://github.com/MPDADDY/capstone_prize-giving_day/assets/62156607/a124bae8-aa5a-43b5-b382-46aa3cd03f24)

Figma design 👇 : 
![navd](https://github.com/MPDADDY/capstone_prize-giving_day/assets/62156607/3e785639-06f3-4e52-95ea-7a3627bd1f26)